README.md

```diff
@@ -32,6 +32,7 @@ synchronisation between multiple hosts. A sample setup for this can be found [he
 
     © 2016 Christian Pointner <equinox@spreadspace.org>
     © 2016 Markus Grüneis <gimpf@gimpf.org>
+    © 2016 whawty contributors (see AUTHORS file)
```

Comment:
because I didnt dare to remove the nox' name. also maybe to highlight maintainers vs. contributors. :P but feel free to push to this branch.